### DISCUSSION TITLE
Error in development mode - Remix App

### DISCUSSION THREAD

**Discussion by edbella** [prisma/prisma#23386] [Q&A]
### Question

I recently updated prisma and accelerate extension to the latest version. I cleared my `node_modules` folder before I made this update and I have had issues starting up my remix app ever since.

<img width="1370" alt="Screenshot 2024-03-06 at 09 50 27" src="https://github.com/prisma/prisma/assets/30025478/7c2da61c-b851-4963-b64b-4303e1366ddd">

I also raised the issue on the [Remix Q&A](https://github.com/remix-run/remix/discussions/8989) and hoping to get a resolution soon.
 

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider  = "mongodb"
  url       = env("DATABASE_URL")
  directUrl = env("DIRECT_DATABASE_URL")
}

enum UserCategory {
  ADVISOR
  INVESTOR
  FOUNDER
}

// Model for the user
model User {
  id                 String         @id @map("_id")
  firstName          String
  lastName           String
  country            String
  email              String         @unique
  emailVerified      Boolean        @default(false)
  phoneNumber        String         @unique
  lastSeen           DateTime       @default(now())
  userCategory       UserCategory[]
  userSubCategory    String[]       @default([""])
  authProviders      String[]
  isActive           Boolean        @default(true)
  isIdentityVerified Boolean        @default(false)

  // Defaults
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
}
```

- OS: macOS
- Database: MongoDB
- Node.js version: 20.11.1

- prisma - 5.10.2
- @prisma/extension-accelerate - 1.0.0
- @prisma/client - 5.10.2

**Comment by janpio** [comment]
What was the version of Prisma and extension that used to work fine with your codebase?
Any other changes besides only this version upgrade that made it fail?

**Reply by edbella** [reply]
I've reverted back to the last stable version I was using before this. But the error persists. Its a bug that I can't seem to find a resolution on. Turns out the prisma accelerate extension what was I recently updated

- @prisma/extension-accelerate - 0.6.3
